full/src/test/java/apoc/metrics/MetricsTest.java

```diff
@@ -50,6 +51,7 @@ public static void afterAll() {
     }
 
     @Test
+    @Ignore
```

Comment:
Can you leave TODOs for these tests please? A TODO somehwere is easily searchable, the @Ignore task alone may not give us enough information on whether we need to un-ignore these tests